store/sqlstore/command_store.go

```diff
@@ -73,8 +82,14 @@ func (s SqlCommandStore) Get(id string) (*model.Command, *model.AppError) {
 
 func (s SqlCommandStore) GetByTeam(teamId string) ([]*model.Command, *model.AppError) {
 	var commands []*model.Command
-
-	if _, err := s.GetReplica().Select(&commands, "SELECT * FROM Commands WHERE TeamId = :TeamId AND DeleteAt = 0", map[string]interface{}{"TeamId": teamId}); err != nil {
+	query, args, err := s.getQueryBuilder().
```

Comment:
minor nit: can we use the same naming convention as your other `ToSql()` calls.
```
sql, args, err := s.getQueryBuilder()....
```

Don't mind `sql` or `query`, as long as they're consistent across all the queries below. :)

Reply:
Done. Going with `sql`.